cmd/signozschemamigrator/schema_migrator/logs_migrations.go

```diff
@@ -230,4 +232,198 @@ ORDER BY name ASC`,
 			},
 		},
 	},
+	{
+		MigrationID: 1005,
+		UpItems: []Operation{
+			CreateTableOperation{
+				Database: "signoz_logs",
+				Table:    "path_types",
+				Columns: []Column{
+					{Name: "path", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+					{Name: "type", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+					{Name: "last_seen", Type: ColumnTypeUInt64, Codec: "DoubleDelta, LZ4"},
+				},
+				Indexes: []Index{
+					{Name: "path_minmax", Expression: "path", Type: "minmax", Granularity: 1},
+					{Name: "type_set", Expression: "type", Type: "set(25)", Granularity: 4},
+					{Name: "last_seen_minmax", Expression: "last_seen", Type: "minmax", Granularity: 1},
+				},
+				Engine: ReplacingMergeTree{
+					MergeTree: MergeTree{
+						OrderBy:     "(path, type)",
+						PartitionBy: "toDate(last_seen / 1000000000)",
+						TTL:         "toDateTime(last_seen / 1000000000) + toIntervalSecond(1296000)",
+						Settings: TableSettings{
+							{Name: "index_granularity", Value: "8192"},
+							{Name: "ttl_only_drop_parts", Value: "1"},
+						},
+					},
+				},
+			},
+			CreateTableOperation{
+				Database: "signoz_logs",
+				Table:    "distributed_path_types",
+				Columns: []Column{
+					{Name: "path", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+					{Name: "type", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+					{Name: "last_seen", Type: ColumnTypeUInt64, Codec: "DoubleDelta, LZ4"},
+				},
+				Engine: Distributed{
+					Database:    "signoz_logs",
+					Table:       "path_types",
+					ShardingKey: "cityHash64(path, type)",
+				},
+			},
+			CreateTableOperation{
+				Database: "signoz_logs",
```

Comment:
I might have a generic table for this, will discuss. Since the main goal for this will be to use the new column if the query range is new else use the old one. So might need a generic solution. It's similar to this.